intro/arrays/code.js

```diff
@@ -1,3 +1,34 @@
-
 module.exports = {
+  each: function(callback, items) {
+    for (let i = 0; i < items.length; i++) {
+      callback(items[i])
+    }
+  },
+
+  each_in_reverse: function(callback, items) {
+    for (let i = items.length - 1; i >= 0; i--) {
+      callback(items[i])
+    }
+  },
+
+  map: function(callback, items) {
+    let results = []
+    for (let i = 0; i < items.length; i++) {
```

Comment:
Once you have your each implementation, you don't need to loop anymore. Reimplement using your each method